docker/presets/models/tfs/download_and_push_model.py

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+
+from huggingface_hub import snapshot_download
+
+# Custom media type for OCI artifacts
+CUSTOM_MEDIA_TYPE = "application/vnd.kaito.llm.v1"
```

Comment:
This custom media type won't affect the behavior of oras. It's just a annotation to help user identify what it is.